app/server/models/user.py

```diff
@@ -88,6 +89,7 @@ class User(ManyOrgBase, ModelBase, SoftDelete):
     TFA_enabled = db.Column(db.Boolean, default=False)
     pin_hash = db.Column(db.String())
     seen_latest_terms = db.Column(db.Boolean, default=False)
+    registration_method = db.Column(db.String)
```

Comment:
enum?

Reply:
Wasn't sure if it was a good idea to add it there, I am not opinionated for or against it.
@enjeyw is there a benefit to using Enums on the db as opposed to strings for storing "constants"

Reply:
> is there a benefit to using Enums on the db as opposed to strings for storing "constants"

well it's being done in several places already in similar situations, like "transfer reason"